conans/requirements.txt

```
@@ -6,7 +6,7 @@ PyYAML>=3.11, <6.0
 patch-ng>=1.17.4, <1.18
 fasteners>=0.14.1
 six>=1.10.0,<=1.14.0
-node-semver==0.6.1
+node-semver>=0.6.1, <0.8.0
```

```suggestion
node-semver>=0.6.1
```

@memsharded we shouldn't ignore 0.8.0. Both 0.6.1 and 0.8.0 will result in the same behavior.

---

Yes, but we should probably put the limit on 0.8.0. As it is unstable 0.X, future versions can break. Please add <=0.8.0

---

Okay, @memsharded you can merge it directly by this comment